<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Kevin Rue-Albrecht" />
    <meta name="date" content="2020-08-06" />
    <title>Kevin Rue-Albrecht’s resume</title>

    
        <script src="Kevin-Rue-CV_files/header-attrs-2.3/header-attrs.js"></script>
        <link href="Kevin-Rue-CV_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
        <link href="Kevin-Rue-CV_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
        <link href="Kevin-Rue-CV_files/paged-0.10/css/resume.css" rel="stylesheet" />
        <script src="Kevin-Rue-CV_files/paged-0.10/js/config.js"></script>
        <script src="Kevin-Rue-CV_files/paged-0.10/js/paged.js"></script>
        <script src="Kevin-Rue-CV_files/paged-0.10/js/hooks.js"></script>
        <script src="Kevin-Rue-CV_files/accessible-code-block-0.0.1/empty-anchor.js"></script>
    
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<p><img src="kennedy.jpg" style="width:80.0%" alt="Kevin Rue-Albrecht" /></p>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:kevinrue67@gmail.com" class="email">kevinrue67@gmail.com</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/kevinrue">github.com/kevinrue</a></li>
<li>For more information, please contact me via email.</li>
</ul>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p>Experienced in statistical analysis, genomics, and software engineering.</p></li>
<li><p>Full experience with next generation sequencing data analysis.</p></li>
<li><p>Highly skilled in R, Bash, Python, with experience in C#, JavaScript, HTML, SQL, PHP, CSS, LaTeX, Perl, and Matlab.</p></li>
</ul>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>This resume was made with the R package <a href="https://github.com/rstudio/pagedown"><strong>pagedown</strong></a>.</p>
<p>A PDF version is available <a href="Kevin-Rue-CV.pdf">here</a>.</p>
<p>Last updated on 2020-08-06.</p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Kevin Rue-Albrecht</h2>
<div id="computational-biologist-dipl.-ing.-ph.d." class="section level3">
<h3>Computational Biologist (Dipl. Ing., Ph.D.)</h3>
<p>I am a computational biologist at the <a href="http://www.ox.ac.uk/">University of Oxford</a>.</p>
<p>My research interests in computational biology include software engineering best practices, DevOps, single-cell genomics, and interactive data visualization.
I particularly enjoy using and contributing R packages part of the <a href="https://www.bioconductor.org/">Bioconductor project</a>.
A list of software packages that I maintain or contributed to is available on the “Software” page of my <a href="https://kevinrue.github.io/">website</a>.</p>
<p>My academic research primarily explores the host immune response to infectious diseases, inflammation, and self-antigens.</p>
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="university-college-dublin" class="section level3">
<h3>University College Dublin</h3>
<p>Ph.D. in Computational Infection Biology</p>
<p>Dublin, Ireland</p>
<p>2015</p>
<p><strong>Thesis:</strong> Comparative systems biology analyses of the bovine transcriptional response to species of the <em>Mycobacterium</em> genus.</p>
<ul>
<li><a href="https://www.bioconductor.org/"><em>Bioconductor</em></a> package for <a href="http://geneontology.org/">Gene Ontology (GO)</a> analysis of gene expression data (<img src="bioconductor.favicon.ico" alt="Bioconductor favicon" width="10px"> <a href="https://bioconductor.org/packages/GOexpress/"><code>GOexpress</code></a>).</li>
</ul>
</div>
<div id="école-polytechnique-universitaire-polytech-nice-sophia" class="section level3">
<h3>École Polytechnique Universitaire Polytech Nice-Sophia</h3>
<p>Ingénieur diplômé, MSc</p>
<p>Sophia Antipolis</p>
<p>2011</p>
<div class="concise">
<ul>
<li>Pharmacology</li>
<li>Biotechnology</li>
<li>Toxicology</li>
<li>Environmental Safety</li>
<li>Bioinformatics</li>
</ul>
</div>
<p><strong>Major:</strong> Bioinformatics and Modelling for Biology</p>
</div>
<div id="cpge-bcpst-véto---lycée-jean-rostand" class="section level3">
<h3>CPGE BCPST Véto - Lycée Jean Rostand</h3>
<p>Class Préparatoire aux Grandes Écoles</p>
<p>Strasbourg, France</p>
<p>2008</p>
<div class="concise">
<ul>
<li>Biology</li>
<li>Chemistry</li>
<li>Physics</li>
<li>Earth Sciences</li>
<li>Mathematics</li>
<li>Programming</li>
</ul>
</div>
</div>
</div>
<div id="research-experience" class="section level2" data-icon="laptop">
<h2>Research Experience</h2>
<div id="postdoctoral-researcher---computational-biologist" class="section level3">
<h3>Postdoctoral Researcher - Computational Biologist</h3>
<p><a href="https://www.imm.ox.ac.uk/research/units-and-centres/mrc-wimm-centre-for-computational-biology/groups/sims-group-computational-genomics">Sims Group</a>, <a href="https://www.imm.ox.ac.uk/research/units-and-centres/mrc-wimm-centre-for-computational-biology">MRC WIMM Centre for Computational Biology</a>, <a href="http://www.ox.ac.uk/">University of Oxford</a></p>
<p>Oxford, UK</p>
<p>2020-present</p>
<ul>
<li>Development and delivery of training materials to cohorts of trainees as part of the <a href="https://www.imm.ox.ac.uk/research/units-and-centres/mrc-wimm-centre-for-computational-biology/training/oxford-biomedical-data-science-training-programme">Oxford Biomedical Data Science Training Programme</a>.</li>
<li>Development of novel bioinformatics tools and pipelines following software engineering best practices.</li>
<li>Contribution to scientific reports and publications.</li>
</ul>
</div>
<div id="postdoctoral-researcher---computational-biologist-1" class="section level3">
<h3>Postdoctoral Researcher - Computational Biologist</h3>
<p><a href="https://www.kennedy.ox.ac.uk/research/computational-genomics">Sansom Group</a>, <a href="https://www.kennedy.ox.ac.uk/">Kennedy Institute of Rheumatology</a>, <a href="http://www.ox.ac.uk/">University of Oxford</a></p>
<p>Oxford, UK</p>
<p>2017-2020</p>
<ul>
<li>Molecular pathogenesis of inflammatory bowel disease, in collaboration with the <a href="https://www.kennedy.ox.ac.uk/research/mucosal-immunology">Powrie Group</a>.</li>
<li>Role of thymic epithelial cells in T-cell development, in collaboration with the <a href="https://www.imm.ox.ac.uk/research/units-and-centres/mrc-human-immunology-unit/research-groups/developmental-immunology-research-group">Holländer Group</a> and <a href="https://www.ed.ac.uk/mrc-human-genetics-unit/research/ponting-group">Ponting Group</a>.</li>
<li>Contribution to software pipelines for single-cell genomics data analysis (<i class="fab fa-github"></i> <a href="https://github.com/sansomlab/tenx">sansomlab/tenx</a>).</li>
<li><a href="https://www.bioconductor.org/"><em>Bioconductor</em></a> package for interactive exploration of <a href="https://bioconductor.org/packages/SummarizedExperiment/"><code>SummarizedExperiment</code></a> objects (<img src="bioconductor.favicon.ico" alt="Bioconductor favicon" width="10px"> <a href="https://bioconductor.org/packages/iSEE/"><code>iSEE</code></a>).</li>
<li>R package for the analysis of ChIP-seq data (<i class="fab fa-github"></i> <a href="https://github.com/kevinrue/deeperTools">kevinrue/deeperTools</a>).</li>
</ul>
</div>
<div id="postdoctoral-researcher---computational-biologist-2" class="section level3">
<h3>Postdoctoral Researcher - Computational Biologist</h3>
<p><a href="https://www.ludwig.ox.ac.uk/research/prof-sir-peter-ratcliffe-oxygen-sensing">Ratcliffe group</a>, <a href="https://www.tdi.ox.ac.uk/">Target Discovery Institute</a>, <a href="http://www.ox.ac.uk/">University of Oxford</a></p>
<p>Oxford, UK</p>
<p>2016 - 2017</p>
<ul>
<li>Management and quality control of genomics data for analyses of DNA targeted
resequencing, ChIP-seq, RNA-seq in the context of oxygen sensing and renal cancer.</li>
<li>Prototype of <a href="https://www.meteor.com/">MeteorJS</a> application to manage sequencing data (<i class="fab fa-github"></i> <a href="https://github.com/kevinrue/Seqbook">kevinrue/Seqbook</a>).</li>
</ul>
</div>
<div id="research-associate---bioinformatics-biostatistics" class="section level3">
<h3>Research associate - Bioinformatics &amp; Biostatistics</h3>
<p><a href="https://www.imperial.ac.uk/people/m.wilkins">Prof. Martin Wilkins</a>’s Group, Imperial Centre for Translational and Experimental Medicine, <a href="https://www.imperial.ac.uk/">Imperial College London</a></p>
<p>London, UK</p>
<p>2015 - 2016</p>
<ul>
<li>Integration of genetic variation, proteomic and metabolomics data and associated deep
phenotype data, in collaboration with the <a href="http://www.med.cam.ac.uk/morrell/research/#graf-group">Morrell Group</a>.</li>
<li><a href="https://www.bioconductor.org/"><em>Bioconductor</em></a> package for the analysis of genetic variants (<img src="bioconductor.favicon.ico" alt="Bioconductor favicon" width="10px"> <a href="https://bioconductor.org/packages/TVTB/"><code>TVTB</code></a>).</li>
</ul>
</div>
<div id="ph.d.-student---computational-infection-biology-rotation-3" class="section level3">
<h3>Ph.D. student - Computational Infection Biology, rotation 3</h3>
<p><a href="http://www.ucd.ie/infectbio/infectionbiologyresearchers/drjimogara/home/">Prof. James O’Gara</a>’s Group, UCD Science Centre North</p>
<p>Dublin, Ireland</p>
<p>Apr. 2012 - Jul. 2012</p>
<ul>
<li>Evaluation of a gene candidate underlying <em>Staphylococcus aureus</em> antibiotic
resistance by Sanger sequencing and biofilm assay</li>
</ul>
</div>
<div id="ph.d.-student---computational-infection-biology-rotation-2" class="section level3">
<h3>Ph.D. student - Computational Infection Biology, rotation 2</h3>
<p><a href="https://www.ucd.ie/news/2011/01JAN11/200111-UCD-scientists-receives-President-of-Ireland-Young-Researcher-Award-PIYRA.html">Dr Neil Ferguson</a>’s Group, <a href="https://www.ucd.ie/conway/">UCD Conway Institute</a></p>
<p>Dublin, Ireland</p>
<p>Jan. 2012 - Apr. 2012</p>
<ul>
<li>Expression and purication of a Hepatitis B protein construct for experimental
screening of interacting drug fragments.</li>
</ul>
</div>
<div id="ph.d.-student---computational-infection-biology-rotation-1" class="section level3">
<h3>Ph.D. student - Computational Infection Biology, rotation 1</h3>
<p><a href="http://shieldslab.ucd.ie/">Shields Lab</a>, UCD Complex and Adaptive Systems Laboratory (CASL)</p>
<p>Dublin, Ireland</p>
<p>Sep. 2011 - Jan. 2012</p>
<ul>
<li><em>In silico</em> structure-based prediction of <em>Mycobacterium bovis</em> epitopes in cattle (supervisor: <a href="https://www.rcsi.com/people/profile/AChubb">Dr. Anthony Chubb</a>).</li>
</ul>
</div>
<div id="research-assistant---computational-biology-intern" class="section level3">
<h3>Research Assistant - Computational Biology, intern</h3>
<p><a href="http://shieldslab.ucd.ie/">Shields Lab</a>, <a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>Jun. 2010 - Sep. 2010</p>
<ul>
<li>Computational analysis of structural disorder in <em>Saccharomyces cerevisiae</em> interacting proteins.</li>
</ul>
</div>
<div id="research-assistant-intern" class="section level3">
<h3>Research Assistant, intern</h3>
<p>INSERM, Unité Mixte de Recherche S725, Biologie des Cellules Dendritiques
Humaines</p>
<p>Strasbourg, France</p>
<p>Jul. 2009 - Aug. 2009</p>
<ul>
<li>Recombinant protein expression of MHC class II molecules in HeLa cells.</li>
</ul>
</div>
</div>
<div id="professional-experience" class="section level2" data-icon="suitcase">
<h2>Professional Experience</h2>
<div id="software-developer-intern" class="section level3">
<h3>Software developer, intern</h3>
<p><a href="http://medit-pharma.com/">MEDIT S.A.</a></p>
<p>Palaiseau, France</p>
<p>Apr. 2011 - Jul. 2011</p>
<ul>
<li>Implementation of structurally-constrained multiple alignment of protein sequences for
the commercial software <a href="http://medit-pharma.com/index.php?page=med-sumo">MED-SuMo</a>.</li>
</ul>
</div>
</div>
<div id="teaching-experience" class="section level2" data-icon="chalkboard-teacher">
<h2>Teaching Experience</h2>
<div id="intuitive-interactive-data-exploration-with-isee" class="section level3">
<h3>Intuitive interactive data exploration with iSEE</h3>
<p><a href="https://www.bdi.ox.ac.uk/">Swiss Institute of Bioinformatics</a></p>
<p>Zoom (online)</p>
<p>2020</p>
<p>Co-organizer of a <a href="https://github.com/iSEE/iSEEWorkshop2020">2h30 workshop</a> at the <a href="https://www.sib.swiss/events/sibdays2020/">SIB Days 2020</a>.</p>
</div>
<div id="r-code-clinic" class="section level3">
<h3>R Code Clinic</h3>
<p><a href="https://www.sib.swiss/">Big Data Institute</a>, <a href="http://www.ox.ac.uk/">University of Oxford</a></p>
<p>Oxford, UK</p>
<p>2019 - present</p>
<p>Volunteer to sit with individuals and assist them with any R problems they need help with.
Website: <a href="https://bdicodeclub.netlify.com/" class="uri">https://bdicodeclub.netlify.com/</a>.</p>
</div>
<div id="interactive-visualisation-of-summarizedexperiment-objects-using-isee" class="section level3">
<h3>Interactive visualisation of SummarizedExperiment objects using iSEE</h3>
<p><a href="http://bioc2019.bioconductor.org/">Bioconductor conference 2019</a></p>
<p>New York, USA</p>
<p>2019</p>
<p>Co-instructor of a <a href="https://github.com/iSEE/iSEEWorkshop2019">2-hour workshop</a> at the <a href="http://bioc2019.bioconductor.org/">Bioconductor conference 2019</a>.</p>
</div>
<div id="molecular-genetics-and-biotech" class="section level3">
<h3>Molecular Genetics and Biotech</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2014</p>
<p>Teaching assistant of <a href="https://sisweb.ucd.ie/usis/!W_HU_MENU.P_PUBLISH?p_tag=MODULE&amp;MODULE=BMOL20090">BMOL20090</a> at <a href="https://www.ucd.ie/">University College Dublin</a>.</p>
</div>
<div id="agricultural-microbiology" class="section level3">
<h3>Agricultural Microbiology</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2014</p>
<p>Teaching assistant of <a href="https://sisweb.ucd.ie/usis/!W_HU_MENU.P_PUBLISH?p_tag=MODULE&amp;MODULE=MICR20010">MICR20010</a> at <a href="https://www.ucd.ie/">University College Dublin</a>.</p>
</div>
</div>
<div id="funding-and-awards" class="section level2" data-icon="medal">
<h2>Funding and Awards</h2>
<div id="rstudio-shiny-contest" class="section level3">
<h3>RStudio Shiny Contest</h3>
<p><a href="https://blog.rstudio.com/2019/04/05/first-shiny-contest-winners#winners">Winner</a> of the “Most technically impressive” prize.</p>
<p>N/A</p>
<p>2019</p>
</div>
<div id="bioconductor-2019-conference" class="section level3">
<h3>Bioconductor 2019 conference</h3>
<p>Conference fee waiver and travel grant for <a href="http://bioc2019.bioconductor.org/">BioC 2019</a>.</p>
<p>New York, USA</p>
<p>2019</p>
<ul>
<li>Instructor of a <a href="https://github.com/iSEE/iSEEWorkshop2019">2-hour workshop</a> at the <a href="http://bioc2019.bioconductor.org/">Bioconductor conference 2019</a>.</li>
<li>Co-organiser of the Special Interest Group (SIG) <a href="https://github.com/Bioconductor/BioC2019/issues/25">Extending gene set and signature representations</a>.</li>
</ul>
</div>
<div id="bioconductor-2017-conference" class="section level3">
<h3>Bioconductor 2017 conference</h3>
<p>Conference fee waiver and travel grant for <a href="https://bioconductor.org/help/course-materials/2017/BioC2017/">BioC 2017</a>.</p>
<p>Boston, USA</p>
<p>2017</p>
<ul>
<li>Flash presentation for <img src="bioconductor.favicon.ico" alt="Bioconductor favicon" width="10px"> <a href="https://bioconductor.org/packages/TVTB/"><code>TVTB</code></a>.</li>
<li>Flash presentation for <img src="bioconductor.favicon.ico" alt="Bioconductor favicon" width="10px"> <a href="https://bioconductor.org/packages/GOexpress/"><code>GOexpress</code></a>.</li>
</ul>
</div>
<div id="ismbeccb-conference" class="section level3">
<h3>ISMB/ECCB conference</h3>
<p>Conference fee waiver</p>
<p>Dublin, Ireland</p>
<p>2015</p>
<p><a href="https://www.iscb.org/ismbeccb2015">23rd Annual International Conference on Intelligent Systems for Molecular Biology (ISMB) and 14th European Conference on Computational Biology (ECCB)</a>.
Special Interest Group (SIG) <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004691">Bioinformatics Open Source Conference (BOSC)</a>.</p>
</div>
<div id="th-annual-phd-symposium-in-computational-biology-innovation" class="section level3">
<h3>5th Annual PhD Symposium in Computational Biology &amp; Innovation</h3>
<p>Best oral presentation</p>
<p>Dublin, Ireland</p>
<p>2014</p>
</div>
<div id="wellcome-trust-four-year-phd-programme" class="section level3">
<h3>Wellcome Trust Four-year PhD Programme</h3>
<p>Computational Infection Biology</p>
<p>Dublin, Ireland</p>
<p>2011</p>
<p><strong>Title:</strong> Transcriptome analysis of the bovine macrophage response to <em>Mycobacterium tuberculosis</em> complex strains.</p>
</div>
</div>
<div id="organiser-and-chair" class="section level2" data-icon="calendar">
<h2>Organiser and Chair</h2>
<div id="special-interest-group" class="section level3">
<h3>Special Interest Group</h3>
<p><a href="http://bioc2019.bioconductor.org/">Bioconductor conference 2019</a>.</p>
<p>New York, USA</p>
<p>2019</p>
<p><strong>Topic:</strong> <a href="https://github.com/Bioconductor/BioC2019/issues/25">Extending gene set and signature representations</a>.</p>
</div>
<div id="genomics-forum" class="section level3">
<h3>Genomics Forum</h3>
<p><a href="https://www.kennedy.ox.ac.uk/">Kennedy Institute of Rheumatology</a>, <a href="http://www.ox.ac.uk/">University of Oxford</a></p>
<p>Oxford, UK</p>
<p>2018-2020</p>
<p>Coordinator of a weekly meeting to present and discuss ongoing projects in the institute.</p>
</div>
<div id="bioinformatics-seminar" class="section level3">
<h3>Bioinformatics Seminar</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2013 - 2014</p>
<p>Co-organiser of a weekly meeting to present and discuss ongoing projects in the university.</p>
</div>
<div id="computational-biology-and-innovation-phd-symposium" class="section level3">
<h3>Computational Biology and Innovation PhD Symposium</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2012</p>
<p>Co-organiser in charge of sponsorships and the abstract book.</p>
</div>
</div>
<div id="training-and-courses" class="section level2" data-icon="leanpub">
<h2>Training and Courses</h2>
<div id="machine-learning" class="section level3">
<h3>Machine Learning</h3>
<p><a href="https://www.coursera.org/learn/machine-learning">Coursera</a>, <a href="https://www.stanford.edu/">Stanford University</a></p>
<p>N/A</p>
<p>2016</p>
</div>
<div id="bioconductor-for-genomic-data-science" class="section level3">
<h3>Bioconductor for Genomic Data Science</h3>
<p><a href="https://www.coursera.org/learn/bioconductor">Coursera</a>, <a href="https://www.jhu.edu/">Johns Hopkins University</a></p>
<p>N/A</p>
<p>2015</p>
<p>This course is part of the <a href="https://www.coursera.org/specializations/genomic-data-science">Genomic Data Science Specialization</a>.</p>
</div>
<div id="command-line-tools-for-genomic-data-science" class="section level3">
<h3>Command Line Tools for Genomic Data Science</h3>
<p><a href="https://www.coursera.org/learn/genomic-tools">Coursera</a>, <a href="https://www.jhu.edu/">Johns Hopkins University</a></p>
<p>N/A</p>
<p>2015</p>
<p>This course is part of the <a href="https://www.coursera.org/specializations/genomic-data-science">Genomic Data Science Specialization</a>.</p>
</div>
<div id="python-for-genomic-data-science" class="section level3">
<h3>Python for Genomic Data Science</h3>
<p><a href="https://www.coursera.org/learn/python-genomics">Coursera</a>, <a href="https://www.jhu.edu/">Johns Hopkins University</a></p>
<p>N/A</p>
<p>2015</p>
<p>This course is part of the <a href="https://www.coursera.org/specializations/genomic-data-science">Genomic Data Science Specialization</a>.</p>
</div>
<div id="statistics-for-genomic-data-science" class="section level3">
<h3>Statistics for Genomic Data Science</h3>
<p><a href="https://www.coursera.org/learn/statistical-genomics">Coursera</a>, <a href="https://www.jhu.edu/">Johns Hopkins University</a></p>
<p>N/A</p>
<p>2015</p>
<p>This course is part of the <a href="https://www.coursera.org/specializations/genomic-data-science">Genomic Data Science Specialization</a>.</p>
</div>
<div id="teaching-in-higher-education" class="section level3">
<h3>Teaching in Higher Education</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2014</p>
</div>
<div id="hot-topics-in-food-and-nutrition-research" class="section level3">
<h3>Hot Topics in Food and Nutrition Research</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2013</p>
</div>
<div id="sequence-data-analysis-training" class="section level3">
<h3>Sequence Data Analysis Training</h3>
<p>Wageningen Institute of Animal Sciences</p>
<p>Wageningen, Netherlands</p>
<p>2013</p>
</div>
<div id="network-analysis-in-systems-biology" class="section level3">
<h3>Network Analysis in Systems Biology</h3>
<p><a href="https://www.coursera.org/learn/network-biology">Coursera</a>, <a href="https://icahn.mssm.edu/">Icahn School of Medicine at Mount Sinai</a></p>
<p>N/A</p>
<p>2013</p>
</div>
<div id="introductory-statistics-using-r-for-computational-biologists" class="section level3">
<h3>Introductory Statistics using R for Computational Biologists</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2012</p>
</div>
<div id="python-programming-for-computational-biologists" class="section level3">
<h3>Python Programming for Computational Biologists</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2012</p>
</div>
<div id="online-research-skills-for-computational-biologists" class="section level3">
<h3>Online Research Skills for Computational Biologists</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2012</p>
</div>
<div id="advances-in-infection-biology" class="section level3">
<h3>Advances in Infection Biology</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2012</p>
</div>
<div id="bioinformatics-research-seminars" class="section level3">
<h3>Bioinformatics Research Seminars</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2012</p>
</div>
<div id="genomics-principles-and-practical-applications" class="section level3">
<h3>Genomics – Principles and Practical Applications</h3>
<p><a href="https://www.ucd.ie/">University College Dublin</a></p>
<p>Dublin, Ireland</p>
<p>2012</p>
</div>
</div>
<div id="professional-memberships" class="section level2" data-icon="id-badge">
<h2>Professional memberships</h2>
<div id="international-society-for-computational-biology-iscb" class="section level3">
<h3>International Society for Computational Biology (ISCB)</h3>
<p><a href="https://www.iscb.org/" class="uri">https://www.iscb.org/</a></p>
<p>N/A</p>
<p>2012 - present</p>
</div>
</div>
<div id="peer-review" class="section level2" data-icon="newspaper">
<h2>Peer review</h2>
<div id="oup-bioinformatics" class="section level3">
<h3>OUP Bioinformatics</h3>
<p><a href="https://academic.oup.com/bioinformatics" class="uri">https://academic.oup.com/bioinformatics</a></p>
<p>N/A</p>
<p>2019</p>
<p>1 article</p>
</div>
<div id="journal-of-open-source-software" class="section level3">
<h3>Journal of Open Source Software</h3>
<p><a href="https://joss.theoj.org/" class="uri">https://joss.theoj.org/</a></p>
<p>N/A</p>
<p>2019</p>
<p>1 article</p>
</div>
</div>
<div id="selected-publications-and-posters" class="section level2" data-icon="file">
<h2>Selected Publications and Posters</h2>
<div id="orchestrating-single-cell-analysis-with-bioconductor." class="section level3">
<h3>Orchestrating single-cell analysis with Bioconductor.</h3>
<p>Nat Methods.</p>
<p>N/A</p>
<p>2019</p>
<p>Amezquita, R. A., A. T. L. Lun, E. Becht, V. J. Carey, L. N. Carpp, L. Geistlinger, F. Marini, <strong>K. Rue-Albrecht</strong>, D. Risso, C. Soneson, L. Waldron, H. Pages, M. L. Smith, W. Huber, M. Morgan, R. Gottardo and S. C. Hicks</p>
</div>
<div id="the-short-chain-fatty-acid-butyrate-imprints-an-antimicrobial-program-in-macrophages" class="section level3">
<h3>The Short Chain Fatty Acid Butyrate Imprints an Antimicrobial Program in Macrophages</h3>
<p>Immunity.</p>
<p>N/A</p>
<p>2019</p>
<p>Schulthess, J., S. Pandey, M. Capitani, <strong>K. C. Rue-Albrecht</strong>, I. Arnold, F. Franchini, A. Chomka, N. E. Ilott, D. G. W. Johnston, E. Pires, J. McCullagh, S. N. Sansom, C. V. Arancibia-Carcamo, H. H. Uhlig and F. Powrie</p>
</div>
<div id="isee-interactive-summarizedexperiment-explorer." class="section level3">
<h3>iSEE: Interactive SummarizedExperiment Explorer.</h3>
<p>F1000Res.</p>
<p>N/A</p>
<p>2018</p>
<p><strong>Rue-Albrecht, K.</strong>, F. Marini, C. Soneson and A. T. L. Lun</p>
</div>
</div>
<div id="conference-presentations" class="section level2" data-icon="file-powerpoint">
<h2>Conference presentations</h2>
<div id="bioconductor" class="section level3">
<h3>Bioconductor</h3>
<p>Oral presentations.</p>
<p>New York, USA</p>
<p>2019</p>
<ul>
<li><a href="https://kevinrue.github.io/slides/bioc2019-flashtalk/#/">Talk</a>: Interactive and reproducible visualization of <code>SummarizedExperiment</code> objects.</li>
<li><a href="http://biocworkshops2019.bioconductor.org.s3-website-us-east-1.amazonaws.com/page/iSEEWorkshop2019__iSEE-lab/">Workshop</a>: Interactive visualization of <code>SummarizedExperiment</code> objects with <code>iSEE</code>.</li>
<li><a href="https://github.com/Bioconductor/BioC2019/issues/25">Special Interest Group</a>: Extending gene set and signature representations.</li>
</ul>
</div>
<div id="genome-informatics" class="section level3">
<h3>Genome Informatics</h3>
<p>Poster.</p>
<p>Cambridge, UK</p>
<p>2018</p>
<p><strong>Title:</strong> Promiscuous expression of lincRNAs in medullary thymic epithelial cells</p>
</div>
<div id="ngs-sig---single-cell-rna-seq" class="section level3">
<h3>NGS-SIG - Single-cell RNA-seq</h3>
<p>Oral presentation.</p>
<p>Oxford, UK</p>
<p>2017</p>
<p><strong>Title:</strong> Variability of human dendritic cells responses to differentially invasive Salmonella strains at single-cell level.</p>
</div>
<div id="bioconductor-1" class="section level3">
<h3>Bioconductor</h3>
<p>Oral presentations.</p>
<p>Boston, USA</p>
<p>2017</p>
<ul>
<li><code>GOexpress</code>: Visualise and summarise gene expression data using Gene Ontology</li>
<li><code>TVTB</code>: The VCF Tool Box: an effort to summarise and visualise variants</li>
</ul>
</div>
<div id="quantitative-genomics" class="section level3">
<h3>Quantitative Genomics</h3>
<p>Poster.</p>
<p>London, UK</p>
<p>2015</p>
<p><strong>Title:</strong> <code>GOexpress</code>: A R/Bioconductor package for the identification and visualisation of robust gene ontology signatures through supervised learning of gene expression data.</p>
</div>
<div id="bosc---ismbeccb-conference" class="section level3">
<h3>BOSC - ISMB/ECCB conference</h3>
<p>Oral presentation and poster.</p>
<p>Dublin, Ireland</p>
<p>2015</p>
<ul>
<li><a href="https://f1000research.com/slides/4-248">Talk</a> <code>GOexpress</code>: A R/Bioconductor package for the identification and visualisation of robust gene ontology signatures through supervised learning of gene expression data.</li>
<li><a href="https://f1000research.com/posters/4-247">Poster</a> <code>GOexpress</code>: Identify and visualise gene expression using supervised learning and Gene Ontology.</li>
</ul>
</div>
<div id="society-for-general-microbiology-sgm" class="section level3">
<h3>Society for General Microbiology (SGM)</h3>
<p>Poster.</p>
<p>Dublin, Ireland</p>
<p>2014</p>
<p><strong>Title:</strong> Microarray analysis of the bovine macrophage response to <em>Mycobacterium bovis</em>, <em>M. bovis</em> Bacille Calmette-Guérin (BCG) and <em>M. avium</em> subspecies <em>paratuberculosis</em>.</p>
</div>
<div id="virtual-institute-of-bioinformatics-evolution-vibe" class="section level3">
<h3>Virtual Institute of Bioinformatics &amp; Evolution (VIBE)</h3>
<p>Oral presentation.</p>
<p>Carlow, Ireland</p>
<p>2014</p>
<p><strong>Title:</strong> The quest for meaningful visualisation of genome-wide expression data.</p>
</div>
<div id="embo-conference-microbiology-after-the-genomics-revolution---genomes-2014" class="section level3">
<h3>EMBO conference: Microbiology after the genomics revolution - Genomes 2014</h3>
<p>Poster.</p>
<p>Paris, France</p>
<p>2014</p>
<p><strong>Title:</strong> Transcriptome analysis reveals differential innate immune response to members of the <em>Mycobacterium tuberculosis</em> complex.</p>
</div>
<div id="the-acid-fast-club-summer-meeting" class="section level3">
<h3>The Acid Fast Club, Summer Meeting</h3>
<p>Oral presentation.</p>
<p>Berlin, Germany</p>
<p>2014</p>
<p><strong>Title:</strong> Transcriptome analysis reveals differential innate immune response of bovine macrophages to strains of the <em>Mycobacterium tuberculosis</em> complex.</p>
</div>
<div id="ucd-conway-festival-of-research-innovation" class="section level3">
<h3>UCD Conway Festival of Research &amp; Innovation</h3>
<p>Poster.</p>
<p>Dublin, Ireland.</p>
<p>2014</p>
<p><strong>Title:</strong> Transcriptome analysis reveals differential innate immune response to members of the <em>Mycobacterium tuberculosis</em> complex.</p>
</div>
<div id="wellcome-trust-final-year-phd-students-meeting" class="section level3">
<h3>Wellcome Trust Final Year PhD Students’ Meeting</h3>
<p>Poster.</p>
<p>London, UK</p>
<p>2014</p>
<p><strong>Title:</strong> Transcriptome analysis reveals differential innate immune response to members of the <em>Mycobacterium tuberculosis</em> complex.</p>
</div>
<div id="association-of-veterinary-teachers-and-research-workers-avtrw" class="section level3">
<h3>Association of Veterinary Teachers and Research Workers (AVTRW)</h3>
<p>Oral presentation.</p>
<p>Hillsborough, Ireland</p>
<p>2013</p>
<p><strong>Title:</strong> Systems Biology of Host-Pathogen Interactions and Bovine Tuberculosis - Differential transcriptional response of bovine monocyte-derived macrophages following different mycobacterial infections</p>
</div>
<div id="animal-health-ireland-workshop-conference" class="section level3">
<h3>Animal Health Ireland Workshop &amp; Conference</h3>
<p>Poster.</p>
<p>Rochestown Park Hotel, Cork</p>
<p>2013</p>
<p><strong>Title:</strong> Microarray analysis of the bovine macrophage response to <em>Mycobacterium bovis</em>, <em>M. bovis</em> Bacille Calmette-Guérin (BCG) and <em>M. avium</em> subspecies <em>paratuberculosis</em>.</p>
</div>
<div id="virtual-institute-of-bioinformatics-and-evolution-vibe" class="section level3">
<h3>Virtual Institute of Bioinformatics and Evolution (VIBE)</h3>
<p>Oral presentation.</p>
<p>Galway, Ireland</p>
<p>2013</p>
<p><strong>Title:</strong> Systems Biology of Host-Pathogen Interactions and Bovine Tuberculosis - Differential transcriptional response of bovine monocyte-derived macrophages following different mycobacterial infections.</p>
</div>
<div id="computational-biology-innovation-symposium" class="section level3">
<h3>Computational Biology &amp; Innovation Symposium</h3>
<p>Poster.</p>
<p>Dublin, Ireland</p>
<p>2013</p>
<p><strong>Title:</strong> Microarray analysis of the bovine macrophage response to <em>Mycobacterium bovis</em>, <em>M. bovis</em> Bacille Calmette-Guérin (BCG) and <em>M. avium</em> subspecies <em>paratuberculosis</em>.</p>
</div>
<div id="dublin-academy-of-pathogenomics-infection-biology-dapi" class="section level3">
<h3>Dublin Academy of Pathogenomics &amp; Infection Biology (DAPI)</h3>
<p>Poster.</p>
<p>Dublin, Ireland</p>
<p>2013</p>
<p><strong>Title:</strong> Microarray analysis of the bovine macrophage response to <em>Mycobacterium bovis</em>, <em>M. bovis</em> Bacille Calmette-Guérin (BCG) and <em>M. avium</em> subspecies <em>paratuberculosis</em>.</p>
</div>
</div>
<div id="language" class="section level2" data-icon="language">
<h2>Language</h2>
<div id="french" class="section level3">
<h3>French</h3>
<p>Native.</p>
<p>N/A</p>
<p>N/A</p>
</div>
<div id="english" class="section level3">
<h3>English</h3>
<p>Fluent. TOEIC score 970.</p>
<p>N/A</p>
<p>N/A</p>
</div>
<div id="italian" class="section level3">
<h3>Italian</h3>
<p>Working knowledge.</p>
<p>N/A</p>
<p>N/A</p>
</div>
<div id="spanish" class="section level3">
<h3>Spanish</h3>
<p>Limited. Used to be fluent though <i class="far fa-sad-tear"></i>.</p>
<p>N/A</p>
<p>N/A</p>
</div>
<div id="german" class="section level3">
<h3>German</h3>
<p>Limited. Haven’t practiced in a long time.</p>
<p>N/A</p>
<p>N/A</p>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
